azext_edge/edge/_help.py

```diff
@@ -1219,3 +1219,81 @@ def load_iotops_help():
+        "iot ops connector opcua"
+    ] = """
+        type: group
+        short-summary: OPC UA connector management.
+        long-summary: |
+          The connector for OPC UA enables your industrial OPC UA environment to ingress data into
+          your local workloads running on a Kubernetes cluster, and into your cloud workloads.
+          See the following resource for more info https://aka.ms/overview-connector-opcua-broker
+    """
+
+    helps[
+        "iot ops connector opcua trust"
+    ] = """
+        type: group
+        short-summary: Manage trusted certificates for the OPC UA Broker.
+        long-summary: |
+          The trusted certificate list contains the certificates of all the OPC UA servers that the connector
+          for OPC UA trusts. If the connector for OPC UA trusts a certificate authority, it automatically trusts
+          any server that has a valid application instance certificate signed by the certificate authority.
+          See the following resource for more info https://aka.ms/overview-opcua-broker-certificates-management
+    """
+
+    helps[
+        "iot ops connector opcua trust add"
+    ] = """
+        type: command
+        short-summary: Add a trusted certificate to the OPC UA Broker's trusted certificate list.
+        long-summary: |
+            The certificate file extension must be .der or .crt.
+        examples:
+        - name: Add a trusted certificate to the OPC UA Broker's trusted certificate list.
+          text: >
+            az iot ops connector opcua trust add --instance instance --ig instanceresourcegroup --cf certificate_file
+        - name: Add a trusted certificate to the OPC UA Broker's trusted certificate list with custom secret name.
+          text: >
+            az iot ops connector opcua trust add --instance instance --ig instanceresourcegroup --cf certificate_file
+            -s custom_secret_name
+    """
+
+    helps[
+        "iot ops connector opcua issuer"
+    ] = """
+        type: group
+        short-summary: Manage issuer certificates for the OPC UA Broker.
+        long-summary: |
+          The issuer certificate list stores the certificate authority certificates that the connector for OPC UA trusts.
+          This list can be used to manage the trust relationship if OPC UA server's application instance certificate is
+          signed by an intermediate certificate authority, but don't want to automatically trust all the certificates
+          issued by the certificate authority.
+          See the following resource for more info https://aka.ms/overview-opcua-broker-certificates-management
+    """
+
+    helps[
+        "iot ops connector opcua issuer add"
+    ] = """
+        type: command
+        short-summary: Add an issuer certificate to the OPC UA Broker's issuer certificate list.
+        long-summary: |
+            The certificate file extension must be .der, .crt or .crl. When adding a .crl file, a .der or .crt file with
```

Comment:
include an example of this

Reply:
with 2 cmd in one example? do we have example for that?

Reply:
updated the example, splited them into 2